other/bootstrap_daemon/tox-bootstrapd-tmpfile.conf

```diff
@@ -0,0 +1 @@
+d /var/run/tox-bootstrapd 0750 tox-bootstrapd tox-bootstrapd -
```

Comment:
There is no need to restrict a directory storing a pid file to `tox-bootstrapd` user:group.